src/kimmdy/kimmdy-yaml-schema.json

```diff
@@ -11,6 +11,12 @@
       "pytype": "bool",
       "default": false
     },
+    "checkpoints": {
```

Comment:
maybe "checkpoint"?

Reply:
or, to be more explicit that this is not a checkpoint to restart from or a name, but a boolean flag, something like "write-checkpoint"

Reply:
had write_checkpoint before. One can not start a continuation from a yml, but I can see this beeing unclear.
 ok, write_checkpoint it is then.